### DISCUSSION TITLE
How do i get to the config file?

### DISCUSSION THREAD

**Discussion by ghost** [Lissy93/dashy#1446] [Help and Support] [answered]
I don't know if this is a stupid question, but the config file is supposed to be saved in the docker container at /public/conf.yml right? I didn't have any volume bind set up and i can't seem to sudo `docker exec -it Dashy /bin/bash` into it not from the CLI not from portainer, what I'm looking to do is make a copy of the config files(s) and set them to ~/.config/Dashy so that its easier integrated into my backup system too and i can configure the YAML files when i want via the CLI.

**Comment by CrazyWolf13** [comment, marked as answer]
Hi 
Looking at the guide: https://dashy.to/docs/deployment
```
docker run -d \
  -p 8080:80 \
  -v ~/my-conf.yml:/app/public/conf.yml \
  --name my-dashboard \
  --restart=always \
  lissy93/dashy:latest
  ```
I highly suggest you add that bind volume, otherwise if you recreate the container your config will ge **gone**

The problem you are facing is that this docker image has no bash installed, `sudo docker exec -it Dashy /bin/sh` should work :)

**Reply by ghost** [reply]
Much appreciated, thank you!
